core/src/test/java/org/frankframework/pipes/ChecksumPipeTest.java

```diff
@@ -21,41 +25,6 @@ public ChecksumPipe createPipe() {
 		return new ChecksumPipe();
 	}
 
-	@Test
-	public void rightChecksumGeneratedMD5() throws Exception {
```

Comment:
Deze testen waren vrij zinloos: check of je een md5 checksumgenerator aanmaakt als je een md5 generator aanmaakt. Qua coverage verliezen we hier ook niets. De enige relevant is `cantCalculate` hieronder wat mij betreft.